docs/scoring.md

```diff
@@ -0,0 +1,39 @@
+# Goal
+
+The goal of this document is to explain how scoring on Lighthouse works, and how to understand what to do to improve your
+Lighthouse score. 
+
+# What metrics does Lighthouse measure?
```

Comment:
Putting on my user first coming to Lighthouse cap, I might expect there to be some explanation of category scores, etc before jumping into performance metrics

Reply:
i think this is addressed, but feel free to reopen if you think this is still missing.